packages/components/src/components/Provider/Provider.tsx

```diff
@@ -1,8 +1,8 @@
 import React from 'react';
-import { ChakraProvider } from '@chakra-ui/react';
+import { ChakraProvider, ChakraProviderProps } from '@chakra-ui/react';
 
 export interface ProviderProps {}
 
-export const Provider: React.FC<ProviderProps> = ({ children }) => {
-  return <ChakraProvider>{children}</ChakraProvider>;
+export const Provider: React.FC<ProviderProps & ChakraProviderProps> = ({ children, ...props }) => {
```

Comment:
Then we don't need to use an intersection here since the type is already tied into `ProviderProps`.

```suggestion
export const Provider: React.FC<ProviderProps> = ({ children, ...props }) => {
```